recipe/meta.yaml

```diff
@@ -31,6 +35,8 @@ test:
         - test -f $PREFIX/lib/libflann_cpp_s.a                                    # [unix]
         - test -f $PREFIX/lib/libflann_cpp.dylib                                  # [osx]
         - test -f $PREFIX/lib/libflann_cpp.so                                     # [linux]
+        - if not exist %PREFIX%\\Library\\lib\\libflann.lib exit 1                # [win]
+        - if not exist %PREFIX%\\Library\\bin\\libflann.dll exit 1                # [win]
```

Comment:
Please add a Windows test requirements section above like [this one]( https://github.com/conda-forge/hdf5-feedstock/blob/master/recipe/meta.yaml#L39-L40 ) from `hdf5`.